runtime/testy/fake_dawn.py

```diff
@@ -50,19 +50,19 @@ def tcp_relay(port):
     s.bind((host, port))
     s.listen(1)
     conn, addr = s.accept()
+    conn.send(msg)
     while True:
-        next_call = time.time()
-        conn.send(msg)
         receive_msg, addr = conn.recvfrom(2048)
+        next_call = time.time()
```

Comment:
i agree with moving these two `next_call` lines together, but should probably swap back up to be immediately after while True. 
Unneccessary diff.